warehouse/oidc/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from warehouse.oidc.interfaces import IOIDCProviderService
+from warehouse.oidc.services import OIDCProviderServiceFactory
+
+
+def includeme(config):
+    config.register_service_factory(
+        OIDCProviderServiceFactory(provider="github"),
```

Comment:
Sorry, this comment either never got saved or was lost in the refactor:

```suggestion
        OIDCProviderServiceFactory(
            provider="github",
            token_request_url="https://token.actions.githubusercontent.com",
        ),
```

Reply:
Ah! Okay, that makes sense now. Will refactor towards that.

Reply:
In this case we don't actually need the `provider` kwarg at all, since it's only being used for logging at this point.

Reply:
```python
    config.register_service_factory(
        OIDCProviderServiceFactory(
            issuer_url="https://token.actions.githubusercontent.com"
        ),
        IOIDCProviderService,
        name="github",
    )
```